File: 1865.py
a = int(input())
lista = []
vezes = 0
for i in range(0, a):
    b, c = input().split()
    if b == 'Thor':
        lista += ['Y']
        vezes += 1
    if b != 'Thor':
        lista += ['N']
        vezes += 1
for j in range(0, vezes):
    print(f'{lista[j]}')

File: 1865.js
const input = require('fs').readFileSync('/dev/stdin', 'utf8');
const lines = input.split('\n');

let a =  parseInt(lines.shift());
let lista = [];
let vezes = 0;
for(var i = 0; i < a; i++){
    let [b, c] = lines.shift().split(" ");
    if(b === 'Thor'){
        lista.push('Y');
        vezes += 1;
    }
    if(b !== 'Thor'){
        lista.push('N');
        vezes += 1;
    }
}
for(var j = 0; j < vezes; j++){
    console.log(`${lista[j]}`);
}
